File: Secretary.API/Models/ServicoCampo.cs
using System;
using System.Collections.Generic;

namespace Secretary.API.Model
{
    public partial class ServicoCampo
    {
        public long Id { get; set; }
        public DateTime DateCreated { get; set; }
        public DateTime ModifiedDate { get; set; }
        public string Ipaddress { get; set; }
        public int? AnoReferencia { get; set; }
        public DateTime DataEntrega { get; set; }
        public DateTime DataReferencia { get; set; }
        public int? Estudos { get; set; }
        public int? FolhetosBrochuras { get; set; }
        public int? Horas { get; set; }
        public int? Livros { get; set; }
        public int? MesReferencia { get; set; }
        public int? Minutos { get; set; }
        public string Observacao { get; set; }
        public long PioneiroId { get; set; }
        public int? Revisitas { get; set; }
        public int? Revistas { get; set; }
        public int? Publicacoes { get; set; }
        public int? VideosMostrados { get; set; }
        public int HorasBetel { get; set; }
        public int CreditoHoras { get; set; }
        public long CongregacaoId { get; set; }
        public long PublicadorId { get; set; }

        public Congregacao Congregacao { get; set; }
        public Pioneiro Pioneiro { get; set; }
        public Publicador Publicador { get; set; }
    }
}

File: Secretary.SPA/src/app/_models/ServicoCampo.ts
import { Publicador } from './Publicador';
import { Congregacao } from './Congregacao';
import { Pioneiro } from './Pioneiro';

export interface ServicoCampo {
  id: number;
  anoReferencia: number;
  dataEntrega: Date;
  dataReferencia: Date;
  estudos: number;
  folhetosBrochuras: number;
  horas: number;
  livros: number;
  mesReferencia: number;
  minutos: number;
  observacao: string;
  revisitas: number;
  revistas: number;
  publicacoes: number;
  videosMostrados: number;
  horasBetel: number;
  creditoHoras: number;
  // ForeignKey
  pioneiroId: number;
  pioneiro: Pioneiro;
  congregacaoId: number;
  congregacao?: Congregacao;
  publicadorId: number;
  publicador?: Publicador;
}
